### DISCUSSION TITLE
What is the meaning of the "coins" field in "mint" object (RequestNextResponse)

### DISCUSSION THREAD

**Discussion by satran004** [CardanoSolutions/ogmios#219] [Q & A] [answered]
There is a  field called "coins" in "mint" object of  RequestNext method's response. I am always getting value 0 for this field.
Any idea in which scenario this field is used ?

![Screenshot 2022-05-22 at 7 28 44 PM](https://user-images.githubusercontent.com/35016438/169693148-0e1227d9-2e6f-4310-8dec-d6314c125f2f.png)

Thanks.

**Comment by KtorZ** [comment, marked as answer]
Hi, this particular one should always be 0 in this situation, otherwise we have a massive problem with Cardano since it means that ADA are actually being created! 

The reason why it's there is simply because it is re-using a more generic structure for representing values. `coins` actually refer to ADA (or more exactly, Lovelace, so 10^6 ADA); In the case of the mint field, it is still a _value_ but it is in practice impossible to find any ADA here, fortunately.

**Reply by satran004** [reply]
Got it. Thanks @KtorZ
